src/uu/mkdir/src/mkdir.rs

```diff
@@ -20,8 +20,6 @@ use uucore::mode;
 use uucore::{display::Quotable, fs::dir_strip_dot_for_creation};
 use uucore::{format_usage, InvalidEncodingHandling};
 
-static DEFAULT_PERM: u32 = 0o755;
```

Comment:
windows is still using this variable ;)

Reply:
I didn't consider supporting windows. Well, how do I test windows behavior?

Reply:
Most of the time, fixing the error in the CI is good enough
If you need to do more complex testing (probably not the case here), you could install a Windows VM https://developer.microsoft.com/windows/downloads/virtual-machines/

Reply:
That's really helpful! I will use it as a reference.